plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -324,26 +324,23 @@ def download(
         :return: downloaded path
         """
 
-        def move_to_target_dir(download_path) -> str:
+        def move_to_target_dir(download_tmp_path) -> str:
             """Move downloaded content to target directory"""
-
-            if download_path.is_file():
-                shutil.copy(download_path, target_dir)
+            if download_tmp_path.is_file():
+                # file is ok!
+                shutil.move(download_tmp_path, target_dir)
                 return str(target_dir)
```

Comment:
does this change anything? move vs copy?

Reply:
@Karrenbelt 
not sure about python implementation:
but move inside same FS should be done as just filename rename, when copy creates a copy.
move should be faster and more space efficient